ndex.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<!-- <html> Question 1 part A, B
+    <head>
+        <script>
+            let userString = window.prompt("Enter your name")
+            window.alert(`Thank you ${userString}`);
+                </script>
+    </head>
+
+    <body>
+        <h1>Pop up String</h1>
+        <p id="page">
+             <script>
+                document.getElementById("page").innerHTML = "Your name is " + userString ;
+                </script>
+        </p>
+    </body>
+</html>
+
+<html> <!--Question 2 part A, B-->
+   <!-- <head>
+   </head>
+   <body>
+       <script>
+        let userNum = window.prompt("Enter a number");
+        if(userNum % 2===0){
```

Comment:
Although this does work and the input is converted to a number, be sure to remember that the input is a string.